libethash-cuda/CUDAMiner.cpp

```diff
@@ -60,9 +60,10 @@ namespace eth
 		}
 
 	protected:
-		virtual bool found(uint64_t const* _nonces) override
+		virtual bool found(uint64_t const* _nonces, uint32_t count) override
```

Comment:
A stream can find multiple valid shares

Reply:
Have you seen it ever happened? We removed this feature, but now I can see some pressure  to bring it back.

Reply:
Plenty of cases where single GPU finds multiple valids, but never checked the nonces to see if they were close enough to each other to belong to the same thread block.